pkg/testcommon/testcommon.go

```diff
@@ -214,6 +215,86 @@ func CreateTmpDir(prefix string) string {
 	return fullPath
 }
 
+// MoveGlobalDdevDir creates a temporary global config directory for DDEV
```

Comment:
Wouldn't this be better as CopyGlobalDdevDir()? We're doing a temp copy for tests, true?

Reply:
It is always hard to come up with a good name, thanks!
I will rename the function.